components/dfplayer.rst

```diff
@@ -111,6 +111,39 @@ Configuration options:
 - **loop** (*Optional*, boolean, :ref:`templatable <config-templatable>`): Repeats playing
   the same track. Defaults to ``false``.
 
+``dfplayer.play_mp3`` Action
+----------------------------
+
+Plays a track inside the folder ``mp3``. Files inside the folder must be numbered from 1 
+to 65535, like ``0001.mp3``, ``0002.mp3``, ... etc.
+The folder name needs to be ``mp3``, placed under the SD card root directory, and the 
+mp3 file name needs to be 4 digits, for example, "0001.mp3", placed under the mp3 folder. 
```

Comment:
If the files must be 4 digits, then its not possible to have 65535, the max would be 9999